wdio.conf.ts

```diff
@@ -0,0 +1,365 @@
+  // These regions are used for the Sauce Labs VM cloud and the Sauce Labs Real Device Cloud.
+  // If you don't provide the region it will default for the `us`
+
+  //
+  // ==================
+  // Specify Test Files
+  // ==================
+  // Define which test specs should run. The pattern is relative to the directory
+  // of the configuration file being run.
+  //
+  // The specs are defined as an array of spec files (optionally using wildcards
+  // that will be expanded). The test for each spec file will be run in a separate
+  // worker process. In order to have a group of spec files run in the same worker
+  // process simply enclose them in an array within the specs array.
+  //
+  // The path of the spec files will be resolved relative from the directory of
+  // of the config file unless it's absolute.
+  //
+  specs: ['./__tests__/e2e/**/*.ts'],
+  // Patterns to exclude.
+  exclude: [
+    // 'path/to/excluded/files'
+  ],
+  //
+  // ============
+  // Capabilities
+  // ============
+  // Define your capabilities here. WebdriverIO can run multiple capabilities at the same
+  // time. Depending on the number of capabilities, WebdriverIO launches several test
+  // sessions. Within your capabilities you can overwrite the spec and exclude options in
+  // order to group specific specs to a specific capability.
+  //
+  // First, you can define how many instances should be started at the same time. Let's
+  // say you have 3 different capabilities (Chrome, Firefox, and Safari) and you have
+  // set maxInstances to 1; wdio will spawn 3 processes. Therefore, if you have 10 spec
+  // files and you set maxInstances to 10, all spec files will get tested at the same time
+  // and 30 processes will get spawned. The property handles how many capabilities
+  // from the same test should run tests.
+  //
+  maxInstances: 10,
+  //
+  // If you have trouble getting all important capabilities together, check out the
+  // Sauce Labs platform configurator - a great tool to configure your capabilities:
+  // https://saucelabs.com/platform/platform-configurator
+  //
+  capabilities: [
+    {
+      browserName: 'chrome',
+      'bstack:options': {
+        os: 'Windows',
+        osVersion: '10',
+      },
+    },
+    {
+      browserName: 'firefox',
+      'bstack:options': {
+        os: 'Windows',
+        osVersion: '10',
+      },
+    },
+    {
+      browserName: 'safari',
+      'bstack:options': {
+        os: 'OS X',
+        osVersion: 'Big Sur',
+      },
+    },
+    {
+      browserName: 'chrome',
+      'bstack:options': {
+        os: 'OS X',
+        osVersion: 'Big Sur',
+      },
+    },
+    {
+      browserName: 'edge',
+      'bstack:options': {
+        os: 'Windows',
+        osVersion: '10',
+      },
+    },
+    {
+      browserName: 'safari',
+      'bstack:options': {
+        deviceName: 'iPhone 14 Pro Max',
+        osVersion: '16',
+      },
```

Comment:
curious, is our mobile app supported on tablets/ipads? (if so, should we add something for ipad/tablet testing)